### PR TITLE
My18

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**

### CODE REVIEW

File: new/PRS13124GabraKrestos.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <revisionDesc>
+            <change who="DN" when="2020-08-04">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ገብረ፡ ክርስቶስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"> Gabra Krǝstos</persName>
```
Comment (eu-genia) [inline new/PRS13124GabraKrestos.xml]: erase blank

File: new/PRS13123TaklaEwostatewos.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ተክለ፡ ኤዎስጣቴዎስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Takla ʾEwosṭātewos</persName>
+                    <persName xml:lang="gez" xml:id="n2">ተክለ፡ ኤዎስጣቴዎስ፡ ወልደ፡ ጐቢስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n2">Takla ʾEwosṭātewos walda Gʷabis</persName>
+                    <persName xml:lang="en" type="normalized" corresp="#n2">Takla ʾEwosṭātewos son of Gʷabis</persName>
```
Comment (eu-genia) [inline new/PRS13123TaklaEwostatewos.xml]: take out normalized (already reviewed though)

File: new/PRS13124GabraKrestos.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/PRS13124GabraKrestos.xml]: done

File: new/PRS13123TaklaEwostatewos.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/PRS13123TaklaEwostatewos.xml]: done

**Pull request merged**